requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy>=1.17.2
 torch>=1.3.1
 packaging
+pesq>=0.0.3
```

Comment:
this shall be in domain depending requirements, but can we also have our own implementation?
cc: @SkafteNicki

Reply:
yes, should be in a new `audio.txt` file in requirements folder.
As the python package is written in c it may be hard to get our own implementation.
So for now I think it is fine to include it as a dependency.

Reply:
done